lib/ramble/ramble/util/stats.py

```diff
@@ -110,4 +131,5 @@ def get_unit(self, unit):
     StatsVar(),
     StatsStdev(),
     StatsCountValues(),
+    StatsCoefficientOfVariation(),
```

Comment:
We iterate over all_stats in generating the results, so putting it at the bottom here will put it last in the list of summary FOMs. Do you think it would make sense to move it up one line so it is grouped with variance and stdev, or is better to be separated?

Reply:
Ah good call, updated!